demisto_sdk/common/hook_validations/release_notes.py

```diff
@@ -0,0 +1,152 @@
+            Option #2:
+                starts with '-' with none or one trailing spaces.
+
+        multi_line_release_notes_regex meaning:
+            Option #1:
+                starts with tab or two to four spaces, then '-' ,then space
+                each line ends with '.'
+            Option #2:
+                starts with '-' with none or one trailing spaces.
+
+        Returns:
+            bool. True if release notes structure valid, False otherwise
+        """
+        one_line_release_notes_regex = r'[a-zA-Z0-9].*\.$|- ?$'
+        multi_line_release_notes_regex = r'(\t+| {2,4})- .*\.$|- ?$'
+
+        release_notes_comments = self.latest_release_notes.split('\n')
+
+        if not release_notes_comments[-1]:
+            release_notes_comments = release_notes_comments[:-1]
+
+        if len(release_notes_comments) == 1:
+            if not re.match(one_line_release_notes_regex, self.latest_release_notes):
+                print_error(F'File {self.release_notes_path} is not formatted according to '
+                            F'release notes standards.\nFix according to {self.LINK_TO_RELEASE_NOTES_STANDARD}')
+                return False
+
+        else:
+            if len(release_notes_comments) <= 1:
+                print_error(F'File {self.release_notes_path} is not formatted according to '
+                            F'release notes standards.\nFix according to {self.LINK_TO_RELEASE_NOTES_STANDARD}')
+                return False
+
+            # if it's one line comment with list
+            if re.match(one_line_release_notes_regex, release_notes_comments[0]):
+                release_notes_comments = release_notes_comments[1:]
+
+            for comment in release_notes_comments:
+                if not re.match(multi_line_release_notes_regex, comment):
+                    print_error(F'File {self.release_notes_path} is not formatted according to '
+                                F'release notes standards.\nFix according to {self.LINK_TO_RELEASE_NOTES_STANDARD}')
+                    return False
+
+        return True
+
+    def validate_file_release_notes_exists(self):
+        """Validate that the file has proper release notes when modified.
+
+        Returns:
+            bool. True if release notes file exists, False otherwise.
+        """
+        # checks that release notes file exists and contains text
+        if not (os.path.isfile(self.release_notes_path) and self.latest_release_notes):
+            print_error(F'File {self.file_path} is missing release notes, '
+                        F'Please add it under {self.release_notes_path}')
+            return False
+
+        return True
+
+    def is_file_valid(self):
+        """Checks if given file is valid.
+
+        Return:
+            bool. True if file's release notes are valid, False otherwise.
+        """
+        # verifying that the other tests are even necessary
+        if not self.validate_file_release_notes_exists():
```

Comment:
Why not add this to the `validations` array?
`all()` will short circuit the tests anyway

Reply:
it won't, I tried it